scripts/pr-audit.sh

```
@@ -0,0 +1,255 @@
+
+  # Build
+  echo -n "  build:     "
+  if npm run build --silent 2>/dev/null; then echo "PASS"; else echo "FAIL"; verify_failures=$((verify_failures + 1)); fi
+
+  # Lint
+  echo -n "  lint:      "
+  if npm run lint --silent 2>/dev/null; then echo "PASS"; else echo "FAIL"; verify_failures=$((verify_failures + 1)); fi
+
+  # Tests + coverage thresholds
+  echo -n "  coverage:  "
+  if npm run test:coverage --silent 2>&1 >/dev/null; then echo "PASS"; else echo "FAIL"; verify_failures=$((verify_failures + 1)); fi
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)